stack_master/scripts/run_zenoh.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+source /opt/ros/humble/setup.bash
+source /ws/install/setup.bash
```

Shellcheck (0.11.0)

[info] Not following: /ws/install/setup.bash was not specified as input (see shellcheck -x).

(SC1091)